tests/test_setup_config_extended.py

```
@@ -306,9 +303,16 @@ def test_setup_downloads_no_selection(mocker, capsys):
     """Test _setup_downloads when user selects nothing."""
     config = {}
 
-    # Mock input to select APK but then menu returns None
-    mocker.patch("builtins.input", side_effect=["a"])  # Choose APK only
-    mocker.patch("fetchtastic.menu_apk.run_menu", return_value=None)
+    # Mock input to select client app but then menu returns None
+    mocker.patch("builtins.input", side_effect=["a"])  # Choose client app only
+
+    def _menu_returns_none():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_menu_returns_none`

Add return type annotation: `None`

(ANN202)